requirements/conda.txt

```diff
@@ -12,7 +12,7 @@ pygments >=2.0
 pylint
 pympler
 pyqt <5.13
-python-language-server >=0.28.2
+python-language-server >=0.28.3
```

Comment:
This needs to be now `>=0.29.0`. Please fix it in all places you touched this value here.